### PR TITLE
IJPS-43 - Update sitting records screen flow, add new sitting records ui added, add new sitting records success added

### PR DESCRIPTION
**Pull request by Josh-HMCTS**
**JIRA link (if applicable)**
https://tools.hmcts.net/jira/browse/IJPS-43

**Change description**
IJPS-43 - Update sitting records screen flow, add new sitting records ui added, add new sitting records success added

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[  x ] No
```

### CODE REVIEW

File: src/app/_services/sitting-records-service/sitting-records.service.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        const headers = {
+            'Content-Type': 'application/json',
+        };
+
+        return this.http.post('/sittingrecord/add', { sittingRecords: newSittingRecords }, { headers: headers});
+    }
+
+
+    createNewSRPostObj(joh: any, tribunalService: any, dateSelected: any, venue: any, period:any) {
+        console.log(joh.johRole)
```
Comment (markdathornehmcts) [inline src/app/_services/sitting-records-service/sitting-records.service.ts:28]: Can anything be done about these warnings?

File: src/app/_workflows/sitting-record-workflow.service.ts [hunk 2]
```diff
@@ -1,5 +1,8 @@
 import { Injectable } from '@angular/core';
 import { FormGroup } from '@angular/forms';
+import { SittingRecordsPostObj, SittingRecordsPostBody } from '../_models/addSittingRecords.model';
```
Comment (markdathornehmcts) [inline src/app/_workflows/sitting-record-workflow.service.ts:3]: If SittingRecordsPostObj is really not used, remove it?

File: src/app/_workflows/sitting-record-workflow.service.ts [hunk 3]
```diff
@@ -10,12 +13,17 @@ import { DateService } from '../_services/date-service/date-service'
...
+  addSittingRecords!: FormGroup;
+  hasVisitedManage = false;
+  cameFromConfirm = false;
+  sittingRecordsRoleList;
+    
   constructor(
+    private dateSvc: DateService,
+    private sittingRecordsSvc: SittingRecordsService,
+    private uInfoSvc: UserInfoService,
     private ViewSittingRecordService: ViewSittingRecordService,
```
Comment (markdathornehmcts) [inline src/app/_workflows/sitting-record-workflow.service.ts:25]: Does this trailing comma need to go?

File: src/app/components/routing/application-routing.component.ts [hunk 4]
```diff
@@ -9,11 +9,11 @@ export class ApplicationRoutingComponent implements OnInit{
 
   constructor(
     private router: Router,
-    private cookies: CookieService
+    private cookies: CookieService,
```
Comment (markdathornehmcts) [inline src/app/components/routing/application-routing.component.ts:12]: Why add the comma?

**Review by markdathornehmcts — APPROVED** [review]
A couple of queries but otherwise LGTM

File: src/app/_workflows/sitting-record-workflow.service.ts (hunk 2 above)
Comment (Josh-HMCTS) [reply src/app/_workflows/sitting-record-workflow.service.ts]: Removed unused import

File: src/app/_services/sitting-records-service/sitting-records.service.ts (hunk 1 above)
Comment (Josh-HMCTS) [reply src/app/_services/sitting-records-service/sitting-records.service.ts]: Updated types of all params

**Pull request merged**